lib/index.ts

```diff
@@ -1,34 +1,40 @@
-import {parseTree} from './parse-mvn';
+import { parseTree } from './parse-mvn';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as subProcess from './sub-process';
+import { PluginResult } from './types';
+import { isJar, getTempPomPathForJar } from './jar';
 
-export function inspect(root, targetFile, options) {
+export async function inspect(root: string, targetFile: string, options): Promise<PluginResult> {
```

Comment:
this is a breaking change, you ok with this?

Reply:
How is it a breaking change?

Reply:
we should use (or extend where necessary) the interfaces defined by https://github.com/snyk/snyk-cli-interface/blob/master/legacy/plugin.ts

Reply:
See: https://github.com/snyk/snyk-mvn-plugin/pull/43 for adding cli-interface

Reply:
Ah it was returning a promise, so the call sites will not change?